x-pack/plugins/security_solution/public/flyout/document_details/preview/footer.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiLink, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import React, { useCallback } from 'react';
+import { i18n } from '@kbn/i18n';
+import { useExpandableFlyoutApi } from '@kbn/expandable-flyout';
+import { FlyoutFooter } from '../../shared/components/flyout_footer';
+import { DocumentDetailsRightPanelKey } from '../shared/constants/panel_keys';
+import { useDocumentDetailsContext } from '../shared/context';
+import { PREVIEW_FOOTER_TEST_ID, PREVIEW_FOOTER_LINK_TEST_ID } from './test_ids';
+
+/**
+ * Footer at the bottom of preview panel with a link to open document details flyout
+ */
+export const PreviewPanelFooter = () => {
+  const { eventId, indexName, scopeId } = useDocumentDetailsContext();
+  const { openFlyout } = useExpandableFlyoutApi();
+
+  const openDocumentFlyout = useCallback(() => {
+    openFlyout({
+      right: {
+        id: DocumentDetailsRightPanelKey,
+        params: {
+          id: eventId,
+          indexName,
+          scopeId,
+        },
+      },
+    });
+  }, [openFlyout, eventId, indexName, scopeId]);
+
+  return (
+    <FlyoutFooter data-test-subj={PREVIEW_FOOTER_TEST_ID}>
+      <EuiFlexGroup justifyContent="center">
+        <EuiFlexItem grow={false}>
+          <EuiLink
+            onClick={openDocumentFlyout}
+            target="_blank"
+            data-test-subj={PREVIEW_FOOTER_LINK_TEST_ID}
+          >
+            {i18n.translate('xpack.securitySolution.flyout.preview.openFlyoutLabel', {
+              defaultMessage: 'Replace with this flyout',
```

Comment:
I think I would ask @nastasha-solomon for the word `Replace`. I know that's what Sergei used but I don't think it makes much sense, I like better the `Open` you had before.

Reply:
@christineweng would you mind adding this to the list of items I need to review in https://github.com/elastic/security-docs/issues/5484? Thanks!

Reply:
@PhilippeOberti it is included in the copy review. The screenshots in https://github.com/elastic/security-docs/issues/5484 reflect both options.

@nastasha-solomon it's in that issue, maybe buried in the texts 😅 
![image](https://github.com/elastic/kibana/assets/18648970/7c083af2-663c-44de-99ef-5d5b0519b43c)

Reply:
@PhilippeOberti FYI i updated copy for files we own. copy updates for entity previews will be in a separate PR.